scootapi/client/run_job_cmd.go

```diff
@@ -66,20 +51,7 @@ func (c *runJobCmd) run(cl *simpleCLIClient, cmd *cobra.Command, args []string)
 		if err != nil {
 			return err
 		}
-		var jsonJob JobDef
-		err = json.Unmarshal(asBytes, &jsonJob)
-		if err != nil {
-			return err
-		}
-
-		jobDef.Tasks = make(map[string]*scoot.TaskDefinition)
-		for taskName, jsonTask := range jsonJob.Tasks {
-			taskDef := scoot.NewTaskDefinition()
-			taskDef.Command = scoot.NewCommand()
-			taskDef.Command.Argv = jsonTask.Args
-			taskDef.SnapshotId = &jsonTask.SnapshotID
-			jobDef.Tasks[taskName] = taskDef
-		}
+		thrifthelpers.JsonDeserialize(jobDef, asBytes)
```

Comment:
Looks like this is reverting Drew's recent change.

Drew changed it to parse a reasonable JSON, not the thrift JSON (which uses numeric field names instead of, y'know, names)

 Is that intentional?